bigdecimal.gemspec

```diff
@@ -51,4 +51,6 @@ Gem::Specification.new do |s|
   end
 
   s.required_ruby_version = Gem::Requirement.new(">= 2.5.0")
+
+  s.metadata["changelog_uri"] = s.homepage + "/blob/master/CHANGES.md"
```

Comment:
@olleolleolle thank you for this suggestion. My feeling is that the link should take the viewer to the latest change log (CHANGES.md). When I am looking to assess the impact of a new version of a gem to our application I want to quickly see what has changed, and ideally have anything impactful highlighted. At times I will be looking at multiple version changes between the one currently in use gem and the latest, so having everything clear to see on one page works best for me.

Reply:
@olleolleolle Referring to the master branch is good because the latest `CHANGES.md` should include all the changes of the released versions. So, I'll merge this as is.